### PR TITLE
fix(exchange): use angular-translate for exchange

### PR DESCRIPTION
**Pull request by neolitec**
SCFRCA-1049

Update ovh-module-exhange to v8+

### CODE REVIEW

File: client/app/app.js [hunk 1]
```diff
@@ -472,68 +477,54 @@ angular
...
-
-                        return $translate.refresh();
-                    }
-
-                    return null;
-                }
-            }); // translation.addResolvable
-        }); // translationProvider.onBefore
-    })
+    // .config(($transitionsProvider) => {
```
Comment (cbourgois) [inline client/app/app.js]: Should be removed

File: client/app/app.js [hunk 2]
```diff
@@ -549,47 +540,87 @@ angular
...
+                modifyThisStep: $translate.instant("common_stepper_modify_this_step"),
+                skipThisStep: $translate.instant("common_stepper_skip_this_step"),
+                nextButtonLabel: $translate.instant("common_stepper_next_button_label"),
+                submitButtonLabel: $translate.instant("common_stepper_submit_button_label")
             };
-
-            removeHook();
         });
+
+        // const removeHook = $transitions.onSuccess({}, () => {
```
Comment (cbourgois) [inline client/app/app.js]: Should be removed

**Comment by neolitec** [comment]
Please help.

**Comment by FredericEspiau** [comment]
@neolitec what do you need help about ?

**Comment by jleveugle** [comment]
Hello @neolitec,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**